packages/lib-react-components/src/Markdownz/Markdownz.spec.js

```diff
@@ -11,4 +13,257 @@ describe('<Markdownz />', function () {
   })
 
   it('renders without crashing', function () { })
+
+  it('renders markdown', function () {
+    expect(wrapper.find('h1')).to.have.lengthOf(1)
+    expect(wrapper.html().includes('Zooniverse')).to.be.true
+  })
+
+  describe('#buildResourceURL', function () {
+    let buildResourceURLSpy
+    before(function () {
+      buildResourceURLSpy = sinon.spy(Markdownz.prototype, 'buildResourceURL')
+    })
+
+    afterEach(function () {
+      buildResourceURLSpy.resetHistory()
+    })
+
+    after(function () {
+      buildResourceURLSpy.restore()
+    })
+
+    it('should return an empty string if symbol is not defined', function () {
+      const url = wrapper.instance().buildResourceURL('srallen')
+      expect(url).to.equal('')
+    })
+
+    it('should return an empty string if the resource is falsy, but the mention symbol is', function () {
+      const url = wrapper.instance().buildResourceURL('', '@')
+      expect(url).to.equal('')
+    })
+
+    describe('when the symbol is #', function () {
+      it('should return the expected url with a search query param', function () {
+        const resource = 'tigers'
+        const baseURL = ''
+        const expectedReturnValue = `${baseURL}/talk/search?query=${resource}`
```

Comment:
This URL runs a text search for `resource`, which is a long-standing bug  that I think has been bothering volunteers. Eg. this URL returns comments that don't include '#transit'.
https://www.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/talk/search?query=transit

Can we change Markdownz to add the # character (%23) to the search?
https://www.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/talk/search?query=%23transit

Reply:
Thanks, I was unaware that was a bug. Returning this URL is still the current functionality of Markdownz, so we should decide if we want to fix it there too since Talk probably won't be updated for a while.

Reply:
I think hashtags have been broken in Talk since it first launched. We can fix it in the existing code if it's a small patch. Otherwise, this could be one reason to migrate Talk across to the new component.